generated/nifake/tests/test_session.py

```diff
@@ -765,12 +765,14 @@ def test_add_properties_to_session_error(self):
             try:
                 session.nonexistent_property = 5
```

Comment:
we need to also test session['4']. session.nonexistent_property

(also, shouldn't this be session.non_existent_property?)

Reply:
Fixed